ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -417,9 +417,9 @@ static List<StorageAccountInfo> parseStorageAccountInfo(String storageAccountInf
 
   public AzureCloudConfig(VerifiableProperties verifiableProperties) {
     azureStorageConnectionString = verifiableProperties.getString(AZURE_STORAGE_CONNECTION_STRING, "");
-    cosmosEndpoint = verifiableProperties.getString(COSMOS_ENDPOINT);
-    cosmosDatabase = verifiableProperties.getString(COSMOS_DATABASE);
-    cosmosCollection = verifiableProperties.getString(COSMOS_COLLECTION);
+    cosmosEndpoint = verifiableProperties.getString(COSMOS_ENDPOINT, ""); // just add a default "" else instantiation fails
```

Comment:
Why do we prefer emtry string than null in this case?

Reply:
Because we don't use cosmos in V2 but the code expects us to initialize it.